include/wabt/wast-parser.h

```diff
@@ -262,7 +262,28 @@ class WastParser {
   Errors* errors_;
   WastParseOptions* options_;
 
-  CircularArray<Token, 2> tokens_;
+  // two-element queue of upcoming tokens
```

Comment:
I'd still rather give this data type a name and define it out-of-line, if only to make the containing class easier to read.

Reply:
Maybe `TokenQueue`?

Reply:
Done.